.gitignore

```diff
@@ -23,6 +23,11 @@ lib
 lib64
 pip
 
+# Build configurations
+kolibri/utils/build_config
+# This can be dynamically changed at build time, and it should never change otherwise
+requirements.txt
```

Comment:
This file is *in* the repository, it should not be edited at build time and ignored.